### PR TITLE
Agenda 20210120

### PR DESCRIPTION
**Pull request by teresanunez**

### CODE REVIEW

**Comment by cpascual** [comment]
sorry if I missed it (I read it quickly) but if it has not been discussed elsewhere, maybe the migration to gitlab should be discussed...

**Comment by teresanunez** [comment]
Thanks, Carlos, I have added that point.

**Review by reszelaz — APPROVED** [review]

**Review by reszelaz — CHANGES_REQUESTED** [review]
@teresanunez last time we agreed to make the meeting on 20.01.2021 Wednesday at 11:00. I don't mind moving to Thursday if all agree (then we could stay on whole Tango webinar planned for Wednesday at 10:00)

**Comment by teresanunez** [comment]
Hi Zibi, it is written at 2021/01/20 also Wednesday. Where have I put it wrong?

**Comment by teresanunez** [comment]
Ah!, in the title. But the agenda is right. Sorry for the confusion

**Review by reszelaz — APPROVED** [review]

**Review by cpt-majkel — APPROVED** [review]

File: 20210120/AGENDA.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+To be held on 2021/01/20 at 11:00 by DESY
+
+## Agenda
+
+1. Discussion about Questionnaire ([#24](https://github.com/sardana-org/sardana-followup/issues/24))
+  - Answers for all institutes?
+  - Visualization of the result?
+2. Discussion about Bug squashing party - Virtual meeting based on questionnaire's results
+3. Urgent user problems/issues - Round table
+3. Review pending points from the previous meeting
```
Comment (reszelaz) [inline 20210120/AGENDA.md]:
```suggestion
4. Review pending points from the previous meeting
```

File: 20210120/AGENDA.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        - [ ] SOLARIS: Cannot stop multiple motors movement in backlash direction [#1421](https://github.com/sardana-org/sardana/issues/1421). Status update
+        - [ ] How to make scans depending on the status of the beam - organize a dedicated meeting to unify the existing solutions (Requirements must be documented) - see discussion started in [#1450](https://github.com/sardana-org/sardana/issues/1450)
+        - [ ] Alba propose examples on how to program without `on_stop()` and `on_abort()` Macro methods.
+        - [ ] MAX IV will provide a docker image to add to CI.
+        - [ ] Shutter integration in continuos scans: status update.
+        - [ ] MaxIV beats configuration status: update from  Alejandro's work.
+        - [ ] Add information on how to access debug information of libtango9-dbgsym (seems like some information is not accessible). 
+        - [ ] Users can break Sardana by messing with configuration in Tango DB, diagnostic script should be created to check that.
+        - [ ] Configure CI on buster (problems with test suite on Python 3.6): status update.
+4. Overview of current developments / PR distribution
```
Comment (reszelaz) [inline 20210120/AGENDA.md]:
```suggestion
5. Overview of current developments / PR distribution
```

File: 20210120/AGENDA.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    - Problems for axis attributes with same name but different properties [#1436](https://github.com/sardana-org/sardana/issues/1436)
+    - Travis is discontinuing the unlimited CI service for OSS projects [#1433](https://github.com/sardana-org/sardana/issues/1433)
+    - Allow SWMR mode and avoid flock problems with NXscanH5_FileRecorder [#1457](https://github.com/sardana-org/sardana/issues/1457)
+    - Decouple attributes default values from memorized values [#1458](https://github.com/sardana-org/sardana/issues/1458)
+    - SEP19
+    - Step scans burst/points_per_step parameter [#1450](https://github.com/sardana-org/sardana/issues/1450)
+    - pre- and post-move hooks not available in mv macro [#1471](https://github.com/sardana-org/sardana/issues/1471)
+5. Migration to gitlab
```
Comment (reszelaz) [inline 20210120/AGENDA.md]:
```suggestion
6. Migration to gitlab
```

File: 20210120/AGENDA.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+6. AOB
```
Comment (reszelaz) [inline 20210120/AGENDA.md]:
```suggestion
6. Sardana release Jan21
   - Who are the release managers?
   - Preselect milestone scope
7. AOB
```

**Review by reszelaz — APPROVED** [review]
Hi @teresanunez,
I just realized that we need to start discussing the Jan21 release. I suggest to add it to the agenda. Also, I corrected the order of points.

**Comment by reszelaz** [comment]
Hi Teresa,
I just realized the agenda for tomorrow misses all the new TODOs from the last meeting. You can easliy find them in the [20201216/MINUTES.md](https://github.com/sardana-org/sardana-followup/blob/master/20201216/MINUTES.md) - these are the "check boxes" that are not in point 6.

**Comment by teresanunez** [comment]
Hi Zibi, thanks, I will try to put it before the meeting ... if not during.

**Review by aureocarneiro — APPROVED** [review]

**Pull request merged**